src/components/dev-dot-content/mdx-components/mdx-table/mdx-table.module.css

```diff
@@ -3,72 +3,59 @@ We are using a wrapper here to enable horizontal scrolling while still retaining
 
 TODO: implement a more accessible horizontal overflow solution as KB-only users aren't able to focus and scroll these elements
 */
-.tableWrapper {
-  margin-bottom: 20px;
-  max-width: 100%;
-  overflow-x: auto;
-  width: 100%;
-
-  & table {
-    background-color: transparent;
-    border-collapse: collapse;
-    border: 1px solid #ddd;
-    border-right: 0;
-    border-spacing: 0;
-    max-width: 100%;
-    width: 100%;
-
-    & th {
-      text-align: left;
-    }
-
-    & td,
-    & th {
-      border-right: 1px solid #ddd;
-      padding: 0;
-    }
-
-    & > tbody > tr:nth-of-type(odd) {
-      background-color: #f9f9f9;
-    }
-
-    & > thead > tr > th,
-    & > tbody > tr > th,
-    & > tfoot > tr > th,
-    & > thead > tr > td,
-    & > tbody > tr > td,
-    & > tfoot > tr > td {
-      padding: 8px;
-      line-height: 1.42857143;
-      vertical-align: top;
-      border-top: 1px solid #ddd;
-    }
 
-    & > thead > tr > th {
-      vertical-align: bottom;
-      border-bottom: 1px solid #ddd;
-    }
-
-    & > caption + thead > tr:first-child > th,
-    & > colgroup + thead > tr:first-child > th,
-    & > thead:first-child > tr:first-child > th,
-    & > caption + thead > tr:first-child > td,
-    & > colgroup + thead > tr:first-child > td,
-    & > thead:first-child > tr:first-child > td {
-      border-top: 0;
-      border-right: 1px solid #ddd;
-    }
-
-    & > tbody + tbody {
-      border-top: 1px solid #ddd;
-    }
-
-    & table {
-      background-color: #fff;
-    }
-
-    & tr.warning td {
-      background: #fcf8e3;
-    }
```

Comment:
💭 Curious how to reproduce the `.warning` class

Reply:
I'm thinking that this is not actually used.
`.warning` would result in a CSS-module-scoped className (like "mdx_table_warning__ag24q"), but nothing in the react component actually specifies `className={s.warning}`